frontend/pages/hubs/[hubUrl].js

```diff
@@ -95,10 +96,39 @@ export default function Hub({
     }
   };
 
+  /**
+   * For example when filtering by location="San Francisco", the url should
+   * automatically change to active filters. This enables filtered searches
+   * to persist, so that they can be easily shareable to other users.
+   *
+   * Builds a URL and updates window state. E.g. something like:
+   * http://localhost:3000/browse?&country=Austria&city=vienna&
+   */
+  const persistFiltersInURL = (activeFilters) => {
```

Comment:
This is being executed twice now if you're on a hub page.
1. The first time when `handleApplyNewFilters` is being called in `BrowseContent`
2. `handleApplyNewFilters` then calls the `applyNewFilters` function in `[hubUrl].js` which executes this function again

I don't think we need this here at all since `[hubUrl].js` and `browse.js` both implement the `BrowseContent` component and `browse.js` also doesn't use this function additionally.